### PR TITLE
react-modal

### PR DESCRIPTION
**Pull request by marudor**

### CODE REVIEW

File: definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+type ReactModal$Props = {
+  isOpen: bool,
+  style?: {
+    content?: Object,
+    overlay?: Object,
+  },
+  appElement?: HTMLElement,
+  onRequestClose?: Function,
```
Comment (jeffmo) [inline definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js]: Looks like there's an [onAfterOpen](https://github.com/reactjs/react-modal/blob/master/lib/components/Modal.js#L34) prop in there, too

Comment (jeffmo) [inline definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js]: Also, looks like [`onRequestClose: (event: Event) => void`](https://github.com/reactjs/react-modal/blob/master/lib/components/ModalPortal.js#L143). 

And [`onAfterOpen: () => void`](https://github.com/reactjs/react-modal/blob/master/lib/components/ModalPortal.js#L81)

File: definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import type React from 'react';
+
+type ReactModal$DefaultProps = {
```
Comment (jeffmo) [inline definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js]: Any specific reason you didn't inline these in the `declare module` below?

File: definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import type React from 'react';
```
Comment (jeffmo) [inline definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js]: Does this work for [libs] files?

File: definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js (hunk 2 above)
Comment (marudor) [inline definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js]: Nope.

File: definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js (hunk 3 above)
Comment (marudor) [inline definitions/react-modal_v1.x.x/flow_>=v0.23.x/react-modal_v1.x.x.js]: Well - tests pass.

**Pull request merged**